Dockerfile

```diff
@@ -13,5 +13,6 @@ FROM openjdk:11-jdk-slim
 ARG path=/usr/app
 WORKDIR ${path}
 COPY --from=build ${path}/target/*.jar ${path}/app.jar
+COPY internship-cloud/*.json ${path}/internship-cloud/internship-project.json
```

Comment:
why again all json files?

Reply:
Fixed. I want to keep "*.json" because now we have to constantly change the file name to the one we need